collector.go

```diff
@@ -81,7 +82,10 @@ func (c metaCollector) Collect(ch chan<- prometheus.Metric) {
 	start := time.Now()
 	defer func() {
 		duration := time.Since(start).Seconds()
-		level.Debug(logger).Log("msg", "Scrape duration", "target", targetName(c.target), "duration", duration)
+		e := level.Debug(logger).Log("msg", "Scrape duration", "target", targetName(c.target), "duration", duration)
```

Comment:
These errors are actually handled by the go-kit logging library, no errors are ever returned.